osu.Game.Rulesets.Catch/Mods/CatchModFloatingFruits.cs

```diff
@@ -14,7 +15,7 @@ public class CatchModFloatingFruits : Mod, IApplicableToDrawableRuleset<CatchHit
     {
         public override string Name => "Floating Fruits";
```

Comment:
I assume the reason why mod names aren't also being localised here is their widespread usage, correct?

Reply:
That is correct.

Reply:
Definitely not in this diff. There are many concerns to figure out with regard to that, including but not limited to how to share translations with web, or how to adapt several dependent projects like osu-tools.

Reply:
What's the issue there? I think we can still reference osu.Resources from the rulesets.

I think if we're going to break `Description`, we should also break `Name`, otherwise we'll end up breaking rulesets twice over: https://github.com/taulazer/tau/blob/master/osu.Game.Rulesets.Tau/Mods/TauModDual.cs

Reply:
I'm not sure we should be localising mod names. They historically haven't been, as they are treated as proper nouns.

Reply:
But you can transliterate proper nouns, as is done in Korean: https://osu.ppy.sh/wiki/ko/Gameplay/Game_modifier/Double_Time (lit. deobeul taim). I'm also seeing whisperings on the Japanese net about ハードロック and ダブルタイム.

Are you sure you want to keep these as latin characters?

Reply:
It's a valid direction, but we need to decide and make sure everyone is on board with it. That shouldn't be in the PR, as I think there will be diverging opinions on it.

Currently they aren't translated in stable (or osu-web to my knowledge).

Reply:
Fair enough.